### PR TITLE
[14.0][MIG] stock_location_localization_text

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@eantones this one is ready

**Comment by mariadforgeflow** [comment]
Added Localization in Location form view because in v14 it wasn't displayed.

File: stock_location_localization_text/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: stock_location_localization_text
+#: model:ir.model.fields,field_description:stock_location_localization_text.field_stock_location__posx
+msgid "Corridor (X)"
+msgstr "Pasillo (X)	"
```
Comment (eantones) [inline stock_location_localization_text/i18n/es.po]: @mariadforgeflow Please remove these spaces

File: stock_location_localization_text/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+#. module: stock_location_localization_text
+#: model:ir.model.fields,field_description:stock_location_localization_text.field_stock_location__display_name
+msgid "Display Name"
+msgstr ""
+
+#. module: stock_location_localization_text
+#: model:ir.model.fields,field_description:stock_location_localization_text.field_stock_location__posz
+msgid "Height (Z)"
+msgstr "Altura (Z)	"
```
Comment (eantones) [inline stock_location_localization_text/i18n/es.po]: These spaces too

File: stock_location_localization_text/views/stock_location_views.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+            <field name="inherit_id" ref="stock.view_location_form" />
+            <field name="arch" type="xml">
+            <xpath expr="//group[@name='additional_info']" position="after">
+                <group string="Localization" name="localization">
+                                        <field name="posx" />
+                                        <field name="posy" />
+                                        <field name="posz" />
+                </group>
+            </xpath>
+            </field>
```
Comment (eantones) [inline stock_location_localization_text/views/stock_location_views.xml]: Can you fix the indentation please?

**Review by eantones — CHANGES_REQUESTED** [review]

**Comment by mariadforgeflow** [comment]
@eantones Done!

**Review by eantones — APPROVED** [review]

**Comment by eantones** [comment]
> @eantones Done!

Thanks! :-)

**Pull request merged**